### ISSUE TITLE
bin file

### ISSUE THREAD

**Issue by junegoo94** [LiDAR-Perception/LiDAR-CS#9]
Is it correct that each bin file has around 1000000000 point clouds?
![Screenshot 2024-09-09 162923](https://github.com/user-attachments/assets/28e82def-ef34-4535-af98-1d54672eae3c)

**Comment by fangjin-cool** [comment]
That's not right. The bin format point cloud can not be directly imported to CloudCompare, you need to load it as 
`xyzi = np.fromfile(pc_path, dtype=np.float32).reshape(-1, 4) `
